content/bn/docs/reference/glossary/control-plane.md

```diff
@@ -0,0 +1,24 @@
+---
+title: কন্ট্রোল প্লেন
+id: control-plane
+date: ২০১৯/০৫/১২
```

Comment:
2019-05-12

Reply:
> 2019-05-12

ok Bhaiya , I am doing it in this format.

Reply:
Hmm, until we localize tags in data files, it will take time (actually, quite a long time). For now, we don't have any plan to localize them. We will continue to use tags in English.